plugin/db/driver.go

```diff
@@ -243,6 +243,10 @@ type MigrationInfo struct {
 	Force bool
 }
 
+// PlaceHolderRegexp is the regexp for placeholder.
+// Refer to https://stackoverflow.com/a/6222235/19075342, but we support '.' for now.
+const PlaceHolderRegexp = `[^\\/?%*:|"<>]+`
```

Comment:
"Placeholder" is a single word:
```suggestion
// PlaceholderRegexp is the regexp for placeholder.
// Refer to https://stackoverflow.com/a/6222235/19075342, but we support '.' for now.
const PlaceholderRegexp = `[^\\/?%*:|"<>]+`
```